packages/sw-runtime-caching/src/lib/request-wrapper.js

```diff
@@ -117,7 +120,16 @@ class RequestWrapper {
     assert.atLeastOne({request});
 
     const cache = await this.getCache();
-    return await cache.match(request, this.matchOptions);
+    const response = await cache.match(request, this.matchOptions);
+
+    // Give the cacheWillMatch callbacks a chance to modify the cached response.
+    // TODO: Right now, we run all the callbacks sequentially, and return the
+    // final result. Instead, it might make sense to enforce a limit of a single
+    // cacheWillMatch callback, and return an error if there are multiple ones.
```

Comment:
These are inline comments, not JSDoc. They don't get run through the Markdown processor or included as part of the published documentation, so Markdown syntax seems unneeded.